chi2/config.ini.example

```diff
@@ -20,6 +21,25 @@ chi_pconcepts=schema.table_name
 chi_pcounts=schema.table_name
 chi_pats=schema.table_name
 
+; SQL snippet that says which patterns in the ontology table correspond to 
+; branch nodes (folder nodes) of interest
+; 'ICD9:___' and 'ICD9:___._' match ICD9 codes down to the first four digits 
+; (5th digit gets too granular and sparse). In UTHSCSA's i2b2, all '[_____]%' 
+; are drug categories (yours is likely different). In UTHSCSA's i2b2, 
+; procedure branch nodes have paths that end in '\A________\' whereas leaf 
+; nodes lack the leading 'A'. Your site is likely different.
+chi_branchnodes=c_basecode like 'ICD9:___' or c_basecode like 'ICD9:___._' or c_name like '[_____]%' or c_fullname like '\i2b2\Procedures\PRC\Meta%\A________\'
+
```

Comment:
What are the main problems with SQL-in-config? This will help evaluate possible alternatives.